### ISSUE TITLE
An error when trying to play multiplayer

### ISSUE THREAD

**Issue by TheLizzard** [TheLizzard/Chess-Engine#19]
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.1008.0_x64__qbz5n2kfra8p0\lib\tkinter\__init__.py", line 1883, in __call__
    return self.func(*args)
  File "main.py", line 146, in menu
    self.start_game(event[1])
  File "main.py", line 269, in start_game
    self.board.start_multiplayer()
  File "C:\Users\TheLizzard\board.py", line 257, in start_multiplayer
    player = Multiplayer(self.board, self.master, colour, self.pieces,
  File "C:\Users\TheLizzard\Players\multiplayer.py", line 30, in __init__
    self.build_connection(ip, self.colour)
  File "C:\Users\TheLizzard\Players\multiplayer.py", line 67, in build_connection
    self.connector = Connector(ip=ip, port=65360, server=colour)
  File "C:\Users\TheLizzard\Networking\connector.py", line 75, in __init__
    self.our_sock.connect((ip, port))
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it
```

**Comment by TheLizzard** [comment]
This issue might not come from Windows but be a logical error. I need to test it some more.

**Comment by TheLizzard** [comment]
It is a logical error in `connector.py` on *line 66*
https://github.com/TheLizzard/Chess-Engine/blob/ea685a39e0d5bbafc55ef0f9d5e0076a754e6412/scr/Networking/connector.py#L66
The server is listening on `"localhost"` but when I changed it to `""` it fixed the connection error. Not both computers can exchange heartbeat packets.

**Comment by TheLizzard** [comment]
Yeay another error that I haven't seen before:
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Program Files\Python3.8\lib\tkinter\__init__.py", line 1883, in __call__
    return self.func(*args)
  File "C:\Downloads\Project\main.py", line 146, in menu
    self.start_game(event[1])
  File "C:\Downloads\Project\main.py", line 269, in start_game
    self.board.start_multiplayer()
  File "C:\Downloads\Project\board.py", line 257, in start_multiplayer
    player = Multiplayer(self.board, self.master, colour, self.pieces,
  File "C:\Downloads\Project\Players\multiplayer.py", line 30, in __init__
    self.build_connection(ip, self.colour)
  File "C:\Downloads\Project\Players\multiplayer.py", line 67, in build_connection
    self.connector = Connector(ip=ip, port=65360, server=colour)
  File "C:\Downloads\Project\Networking\connector.py", line 75, in __init__
    self.our_sock.connect((ip, port))
TimeoutError: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
```

**Comment by TheLizzard** [comment]
Fixed 1 of this collection of bugs

**Comment by TheLizzard** [comment]
I think the temporary fix: 26ea4b5e9bfe53f5b3eb6cc0ab25e34699de8e73 works well enough for a permanent fix.
